Description: avoid feature webpki-tokio
 Required crate webpki-roots is not in Debian
Author:
 Jonas Smedegaard <dr@jones.dk>
 kpcyrd <git@rxv.cc>
Forwarded: not-needed
Last-Update: 2025-03-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -77,6 +77,2 @@
 
-[dependencies.webpki-roots]
-version = "0.25"
-optional = true
-
 [dev-dependencies.hyper]
@@ -130,5 +126 @@
 tokio-runtime = ["hyper/runtime"]
-webpki-tokio = [
-    "tokio-runtime",
-    "webpki-roots",
-]
